src/main/java/invtweaks/InvTweaksHandlerSorting.java

```diff
@@ -113,10 +113,7 @@ public InvTweaksHandlerSorting(Minecraft mc_, InvTweaksConfig config, ContainerS
 
     private static boolean canMergeStacks(ItemStack from, ItemStack to) {
         if(areItemsStackable(from, to)) {
-            if(from.stackSize > from.getMaxStackSize()) {
-                return false;
-            }
-            if(to.stackSize < to.getMaxStackSize()) {
+            if (from.stackSize < from.getMaxStackSize() && to.stackSize < to.getMaxStackSize()) {
```

Comment:
This isn't actually equivalent, if you swap them both to be < it would be
`from.stackSize <= from.getMaxStackSize() && to.stackSize < to.getMaxStackSize()`

Note that it's used for filling up stacks, so the reason we care about from is only if it's oversized (above maximum) -- we can take items from an already-full stack.